policyengine_us/parameters/gov/local/ca/la/dwp/ez_save/eligibility/fpg_limit_increase.yaml

```
@@ -0,0 +1,11 @@
+description: Los Angeles County increases the fpg limit by this percentage under the EZ Save program. 
```

similar descriptions elsewhere
```suggestion
description: The Los Angeles Department of Water and Power limits the EZ Save program to households with income below this percentage of the federal poverty line. 
```